soroban-env-host/src/vm.rs

```diff
@@ -33,6 +36,9 @@ use wasmi::{Caller, StoreContextMut};
 
 impl wasmi::core::HostError for HostError {}
 
+pub const WASM_LINEAR_MEMORY_SIZE_LIMIT_BYTES: usize = 0x200000; // 2MB
+pub const WASM_TABLE_ELEMENTS_LIMIT_COUNT: u32 = 1000;
```

Comment:
I've added two new limits here. I think the linear memory size 2MB should be reasonable? Not sure about the table elements count. Would like some feedback.